src/api/documentation/config.yml

```diff
@@ -0,0 +1,21 @@
+introspection:
+  removeTrailingPeriodFromDescriptions: false
+  schemaFile: ./schema.graphql
+  queryNameStrategy: capitalizeFirst
+  fieldExpansionDepth: 2
+  spectaqlDirective:
+    enable: true
+
+extensions:
+  graphqlScalarExamples: true
+
+info:
+  title: EXIPS GraphQL API Reference
+  description: Export Insurance Policies
+
+servers:
+  - url: https://app-exip-api-staging-001.azurewebsites.net/api/graphql
```

Comment:
can this come from an env?

Reply:
Had a look, opens doesn't seem like there is a nice way to do it. I actually think it'll be better to not expose this URL anyway, i've chagned it to be just `https://`.
Without a URL or at least a `https`, the documentation config fails.